### PR TITLE
there is no six module, no need to import it in wait_for_quit

### PR DESCRIPTION
**Pull request by quanwenli**
Signed-off-by: Wenli Quan <wquan@redhat.com>

id: 1467827

### CODE REVIEW

**Comment by quanwenli** [comment]
This case is for windows VM. will python 3 will be installed in windows VM?

**Comment by hereischen** [comment]
I don't know which kind of Python will be installed in windows VM. But I assume you are using Python 2.x on windows VM.
This code does not work on **Python 2.x**, If you simply removes `from six.moves import input`.

**Comment by quanwenli** [comment]
It works in Windows VM and waiting for till input "exit" or "quit".

**Comment by vivianQizhu** [comment]
@quanwenli It definitely works for python 3, but I guess we need adapt it to python 2 currently, where you will need `raw_input()` instead of `input()`, since `input()` in python 2 expect a valid python pattern, instead of a simple string. Unfortunately python 3 has no `raw_input`. That's why we need `six.moves`.
If you hit issue like no six modules, I suppose better solution is to install it on your environment.

**Comment by quanwenli** [comment]
@vivianQizhu the python2.7 was installed on windows guest not linux guest/host, do you mean we need install six module in windows guest ? if yes, maybe we need it from installation cases?

**Comment by vivianQizhu** [comment]
@quanwenli If this script is running on the guest, then yes, it should be installed in guest, you can either install it in advance during installation or just before you use the script in certain test case.

**Comment by quanwenli** [comment]
@vivianQizhu wait_for_quit.py is used for case openflow_acl_test.accessed_deny_all_target.telnet.Windows which is on private bridge and it's on Windows guest. i mean maybe we do not need to consider the compatibility issue here since we do not sure if python 3.x version will be installed on guest vm in tests. If python 3.x is going to plan installed on windows vm. other python script for windows VM need to update too. So here, I suggest replace with raw_input(), what's your option ?

**Comment by vivianQizhu** [comment]
@quanwenli I will accept to replace it with `raw_input()` only if we can prove that:
1. wait_for_quit.py only used by openflow_acl_test.accessed_deny_all_target.telnet
2. All the windows guest version been tested have python 2.x installed by default

**Comment by quanwenli** [comment]
@vivianQizhu
1.  I grep the avocado-vt and qemu, only case openflow_acl_test.accessed_deny_all_target.telnet uses  wait_for_quit.py
2. the python2.x is installed from software_install_32.bat/software_install_64.bat of winutils.iso. it's not related with difference os. 

I choose 32 and 64 bit windows. they all passed. 
(1/1) Host_RHEL.m7.u5.qcow2.virtio_blk.up.virtio_net.Guest.Win8.i386.1.io-github-autotest-qemu.openflow_acl_test.accessed_deny_all_target.telnet: PASS (583.92 s)
 (1/1) Host_RHEL.m7.u5.qcow2.virtio_blk.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.openflow_acl_test.accessed_deny_all_target.telnet: PASS (183.79 s)

**Comment by vivianQizhu** [comment]
Okay to me, request for @luckyh 's approval.

**Comment by quanwenli** [comment]
@luckyh , the python will be run on windows guest, it will not be affect on python3.

**Comment by luckyh** [comment]
@quanwenli the script is not just used for windows.

**Comment by quanwenli** [comment]
@luckyh  I grep the code, it's only used for case openflow_acl_test.accessed_deny_all_target.telnet.Windows

**Comment by quanwenli** [comment]
@luckyh I have updated, it's passed on python 3 python 2, could you review it ? thanks.

File: shared/scripts/wait_for_quit.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
-from six.moves import input
-
 
 while True:
-    in_put = input("")
+    try:
+        _input = raw_input
+    except NameError:
+        _input = input
```
[the remark this answers is not in the document]
Comment (quanwenli) [reply shared/scripts/wait_for_quit.py]: yes, it's better. I have updated. please review.

**Comment by hereischen** [comment]
Acked-by: Haotong Chen hachen@redhat.com

**Pull request merged**